ee/migrations/0013_remove_deprecated_tags.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 3.2.13 on 2022-06-13 14:05
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("ee", "0012_migrate_tags_v2"),
+    ]
+
+    operations = [
+        migrations.RemoveField(model_name="enterpriseeventdefinition", name="deprecated_tags",),
+        migrations.RemoveField(model_name="enterpriseeventdefinition", name="tags",),
+        migrations.RemoveField(model_name="enterprisepropertydefinition", name="deprecated_tags",),
+        migrations.RemoveField(model_name="enterprisepropertydefinition", name="tags",),
```

Comment:
I don't think we can do this – we still have a [policy of keeping deprecated fields in](https://posthog.com/handbook/engineering/databases/schema-changes#do-not-delete-or-rename-django-models-and-fields). AFAIK this hasn't been made safe yet (particularly for self-hosted deployments which can jump by a few versions).